samples/getallvms.py

```diff
@@ -66,11 +68,18 @@ def main():
 
     args = cli.get_args()
 
+    ssl_options = ssl.create_default_context(purpose=ssl.Purpose.SERVER_AUTH,
```

Comment:
Hi @tianhao64, thanks for your suggestion, as I said I'm a python newbie. Will update my patch (actually i have some other modifications too).


Thanks again!